linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/oval/shared.xml

```diff
@@ -0,0 +1,141 @@
+<def-group>
+  <definition class="compliance" id="password_has_interactive_users" version="1"> 
+    {{{ oval_metadata("Checks /etc/passwd for interactive users.") }}}
+    <criteria operator="AND">
+      <criterion comment="Interactive users exist." test_ref="test_has_interactive_users" negate="true"/>
+    </criteria>
+  </definition>
```

Comment:
Right now the build system doesn't support two definitions per `definitions` per `def-group`:
https://github.com/ComplianceAsCode/content/pull/7183/checks?check_run_id=2964362854#step:6:303

You could incorporate the test and objects to a single definition, or move `password_has_interactive_users` to `shared/checks/oval`.

Reply:
Weird, I will double check that I actually pushed up what I intended to. I had moved all of the items for `password_has_interactive_users` to one of the two checks (as the build system rightly complained when it found both). 
I had had some struggles with the build system initially, and I will probably just move the check out to `shared/checks/oval` instead (as I had seen the other pattern of having the check only in another rule somewhere else)

Reply:
I misunderstood what you wrote. How is it that I was able to build? Was there a corner case that I didn't realize?

Reply:
Nah, sorry, I misled you. I was fooled by the test.
So the build works, and the test is probably complaining about non-sense. I'll take a look at it.

Reply:
I think moving the def out to `shared/checks/oval` makes a lot of sense, so I just pushed a commit to do that.